en/docs/wip/change-developer-portal-password.md

```diff
@@ -1,64 +1,95 @@
-# Change the Developer Portal Password
 
-  You can change your own password for the WSO2 API Manager Developer Portal using the Developer Portal UI.  
-
-Follow the instructions below to change your password: 
-
-1. Sign in to the Developer Portal.    
-
-     `https://<hostname>:9443/devportal`     
-
-     Example: `https://localhost:9443/devportal`    
-
-     Use your username and password to sign in.    
-
-2. Click  on your username at the top right corner and select  **Change Password**.  
-
-     [![Change Developer Portal password User Menu]({{base_path}}/assets/img/learn/change-devportal-password-user-menu-click.png)]({{base_path}}/assets/img/learn/change-devportal-password-user-menu-click.png)    
-
-3. Enter your old password, a new password, re-enter the new password, and thereafter click **SAVE** to submit changes.
-
-     [![Change Developer Portal password Save Clicking]({{base_path}}/assets/img/learn/change-devportal-password-policy-guideline-display.png)]({{base_path}}/assets/img/learn/change-devportal-password-policy-guideline-display.png)
+# Change the Developer Portal Password  
+  
+  You can change your own password for the WSO2 API Manager Developer Portal using the Developer Portal UI.    
+  
+Follow the instructions below to change your password:   
+  
+1. Sign in to the Developer Portal.      
+  
+     `https://<hostname>:9443/devportal`   
+  Example: `https://localhost:9443/devportal`   
+  Use your username and password to sign in.      
+  
+2. Click  on your username at the top right corner and select  **Change Password**.    
+  
+     [![Change Developer Portal password User Menu]({{base_path}}/assets/img/learn/change-devportal-password-user-menu-click.png)]({{base_path}}/assets/img/learn/change-devportal-password-user-menu-click.png)      
+  
+3. Enter your old password, a new password, re-enter the new password, and thereafter click **SAVE** to submit changes.  
+  
+     [![Developer portal password change submit]({{base_path}}/assets/img/learn/change-devportal-password-submiting.png)]({{base_path}}/assets/img/learn/change-devportal-password-submiting.png)  
+  
```

Comment:
Mention here that the password u enter should adhere the patterns/policies as described below..

Reply:
Fixed with: https://github.com/wso2/docs-apim/pull/1512/commits/2858d5787980c02936b3694c4abc44117db694ab